docs/tutorials/intro_tutorial.rst

```diff
@@ -169,148 +206,230 @@ The beginning of both classes looks like this:
-context, time is used in the import as evidenced below with
-``mesa.time.Randomactivation``. You can see the different time classes
-as
-`mesa.time <https://github.com/projectmesa/mesa/blob/main/mesa/time.py>`__.
-
-With that in mind, the model code with the scheduler added looks like
-this:
+model uses, and see whether it changes the model behavior. The details
+pertaining to the scheduler interface can be located the same `mesa
+repo <https://github.com/projectmesa/mesa/blob/main/mesa/time.py>`__.
 
-.. code:: ipython3
-
-    import mesa
+With that in mind, the MoneyAgent code is modified below to visually
+show when a new agent is created. The MoneyModel code is modified by
+adding the RandomActivation method to the model. with the scheduler
+added looks like this:
 
+.. code:: ipython3
 
     class MoneyAgent(mesa.Agent):
         """An agent with fixed initial wealth."""
 
         def __init__(self, unique_id, model):
+            # Pass the parameters to the parent class.
             super().__init__(unique_id, model)
+
+            # Create the agent's variable and set the initial values.
             self.wealth = 1
 
         def step(self):
             # The agent's step will go here.
             # For demonstration purposes we will print the agent's unique_id
-            print("Hi, I am agent " + str(self.unique_id) + ".")
+            print(f"Hi, I am an agent, you can call me {str(self.unique_id)}.")
 
 
     class MoneyModel(mesa.Model):
         """A model with some number of agents."""
 
         def __init__(self, N):
             self.num_agents = N
+            # Scheduler is created and added to the model
             self.schedule = mesa.time.RandomActivation(self)
+
             # Create agents
             for i in range(self.num_agents):
                 a = MoneyAgent(i, self)
+                # Add the agent to the scheduler
                 self.schedule.add(a)
 
         def step(self):
             """Advance the model by one step."""
+
+            # The model's step will go here for now this will call the step method of each agent and print the agent's unique_id
             self.schedule.step()
 
-At this point, we have a model which runs – it just doesn’t do anything.
-You can see for yourself with a few easy lines. If you’ve been working
-in an interactive session, you can create a model object directly.
-Otherwise, you need to open an interactive session in the same directory
-as your source code file, and import the classes. For example, if your
-code is in ``money_model.py``:
+
+Running the Model
+~~~~~~~~~~~~~~~~~
+
+A basic model has now been created. The model can be run by creating a
+model object and calling the step method. The model will run for one
+step and print the unique_id of each agent. The model can be run for
+multiple steps by calling the step method multiple times.
+
+Note: If the script file track has been utilized. The script will be
```

Comment:
Could we add a clarifier here?  Maybe ... 

"If using script files (.py), the script will be..".

but this may not be the best, I am just worried for new coders the term "script file" is not in their vocabulary yet.

Reply:
> Note: If the script file track has been utilized. The script will be
utilized in within the python interactive session in the same directory
as your source code file, and import the ``MoneyModel`` class. Then
create a model object and run it for a few steps.

I find this unclear. I will leave this to you to clarify it.